index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>José Fernández-Caminero's Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+            <img src="resources/ic_menu.svg" alt="Toolbar menu">
+        </nav>
+    </header>
+
+    <section>
+        <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
+
+        <p class="crazyParagraph">I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
+
+        <button>Start collaboration</button>
+
+        <footer>
+            <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
+        </footer>
```

Comment:
- Great job done so far, for aesthetic reasons I'd advise that you add this image in CSS using the `background-image` property, please fix this 🙏.
- There are also some little differences between your site and the design in the figma template. The background image in this section occupies the entire width of the page. Please fix this issue in your site 👍 👍 

Your project 👇:
<img width="171" alt="jose" src="https://user-images.githubusercontent.com/73607512/198152422-db21d793-993d-4382-800b-39bdc5814011.PNG">

Figma template:
<img width="171" alt="jose" src="https://user-images.githubusercontent.com/73607512/198152455-dd08b14d-e5b5-46d8-b771-163f4766ec53.PNG">